CONTRIBUTING.md

```diff
@@ -194,6 +195,14 @@ A Pull Request (PR) is considered to be in a "stale" state when the following co
 
 [Labels](https://github.com/a11yproject/a11yproject.com/labels) allow the project maintainers to quickly sort filter and site [Issues](#reporting-issues) and [Pull Requests](#submitting-pull-requests). They will be added and removed as needed.
 
+### Editing Articles
+
+When editing articles, leave the original publish date and author as-is. Add the following to the front matter:
```

Comment:
Personally I would veer more towards mentioning "updating" rather than editing.  e.g. making some typo fixes wouldn't really constitute someone adding their name as a co-author to the article.  However, making significant content updates (adding new information / removing stale information) would be very appropriate to add a new author's name / updated date.

Reply:
Agreed!